docs/azure/sdk/resource-management.md

```diff
@@ -87,29 +113,29 @@ To reduce the number of clients needed to perform common tasks and the number of
 
 To accomplish this, we're introducing three standard types for all resources in Azure:
 
-#### **[Resource].cs**
+#### **[ResourceName]Resource.cs**
```

Comment:
Should we use "class" to introduce this? Something.cs is like to introduce the source code instead of the library itself
```suggestion
#### **[ResourceName]Resource** class
```

Reply:
agree